internal/checker/terraform/rules/aws/aws_ssm_document_encrypted.rego

```diff
@@ -0,0 +1,60 @@
+# METADATA
+# title: "Verify that all Session Manager data is transmitted securely"
+# description: "When working with critical production data, it is strongly advised to use encryption at rest to prevent unwanted access."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_document
+# custom:
+#   id: CB_TFAWS_087
+#   severity: MEDIUM
+package lib.terraform.CB_TFAWS_087
+
+import future.keywords.in 
+
+
+isvalid(block){
+	block.Type == "resource"
+    block.Labels[_] == "aws_ssm_document"
+}
+
+has_attribute(key, value) {
+    _ = key[value]
+}
+
+resource [resource]{
+    block := pass[_]
+	resource := concat(".", block.Labels)
+} 
+resource [resource]{
+    block := fail[_]
+	resource := concat(".", block.Labels)
+} 
+
+
+fail[block] {
+    block := input[_]
+    isvalid(block)
+    contentStr := block.Attributes.content
+    contentParsed := json.unmarshal(contentStr)
+    not has_attribute(contentParsed.inputs, "kmsKeyId")
```

Comment:
I think adding not to `has_attribute` will cause error if the function returns `null` value